cardano-api/test/Test/Cardano/Api/TxBody.hs

```diff
@@ -85,9 +125,10 @@ normalizeOriginal content =
     , txWithdrawals = normalizeWithdrawals $ txWithdrawals content
     }
 
--- | Normalizations applied to roundtrip result data only
-normalizeRoundtrip :: TxBodyContent ViewTx era -> TxBodyContent ViewTx era
-normalizeRoundtrip content@TxBodyContent{txAuxScripts, txIns, txInsCollateral} =
+-- | Normalizations applied to roundtrip result data
+normalizeContentRoundtrip ::
+  TxBodyContent ViewTx era -> TxBodyContent ViewTx era
```

Comment:
Why this indent here?

Reply:
This doesn't fit in 80 columns.

Reply:
The max is 100 columns

Reply:
If you do indent a type signature then it is indented as follows:
```
normalizeContentRoundtrip 
  :: TxBodyContent ViewTx era -> TxBodyContent ViewTx era
```